radix-engine/src/system/system.rs

```diff
@@ -49,6 +50,22 @@ pub struct SystemService<'a, Y: KernelApi<SystemConfig<V>>, V: SystemCallbackObj
     pub phantom: PhantomData<V>,
 }
 
+enum ActorObjectType {
+    SELF,
```

Comment:
Out of interest, why is `SELF` capitalised? Can we just do `Self`?

Reply:
`Self` is reserved by rust